src/mixins/date-mixin.ts

```diff
@@ -86,4 +87,24 @@ export default class DateMixin extends Vue {
     const dateLocalMs = new Date(date).setHours(0, 0, 0, 0)
     return Math.round((dateLocalMs - todayLocalMs) / this.MS_IN_A_DAY)
   }
+
+  /** Validate the DateTime is within the allowed range */
+  isValidDateTime (dateToValidate: Date, ignoreTime: boolean = false): boolean {
+    if (dateToValidate) {
+      const startDate = new Date()
+
+      // Ignore the time difference if the user has input a Date before time.
+      if (ignoreTime && dateToValidate.getDate() === startDate.getDate()) {
```

Comment:
If you are checking "before" something, shouldn't you have a <= instead of === ?

Reply:
This is comparing two numbers ( 1-31 ) where i want it to be true IF the Date input is the same day as today, just specifically want to look at the days only here.

Reply:
Hmm, wasn't clear (still isn't clear, though I understand it now). How about explaining it as above in the code comments? :)

Reply:
Yes sir, 100%. Great idea!

Reply:
Can you please mention this is in the comment?
> i want it to be true IF the Date input is the same day as today, just specifically want to look at the days only here

Reply:
For sure